### ISSUE TITLE
metallb + frr stops sending bgp updates after node reboot

### ISSUE THREAD

**Issue by xhejtman** [metallb/metallb#1354]
Hello,

MetalLB version: 0.12.1
Kubernetes: 1.21.11 (rke2 branding)
Calico. Dual-stack
Kube-proxy with iptables

I tried to use MetalLB with FRR enabled to support dualstack which seems to be not possible without FRR. It works, but if I reboot some (1-2) nodes in 4 nodes cluster, usually 2 nodes continue sending BGP keep alive messages, but other 2 nodes stop to send anything to the configured peer.

If I reinstall metallb, all nodes send updates again until I reboot some of them. Is this known problem? Or is there any workaround?  Without FRR, it seems to be OK, reboot is no problem and everything is working.

**Comment by fedepaol** [comment]
Looks like a duplicate of https://github.com/metallb/metallb/issues/1292, closing this (and we'll have a look at it)

**Comment by fedepaol** [comment]
Reopening, as it seems to be a different thing.
Collecting all the info here

**Comment by fedepaol** [comment]
The logs https://github.com/metallb/metallb/files/8649926/metallb.zip

**Comment by fedepaol** [comment]
Show running-config: 
```
/ # vtysh -c "show running-config"
Building configuration...

Current configuration:
!
frr version 7.5.1_git
frr defaults traditional
hostname kub-as1.priv.cerit-sc.cz
log file /etc/frr/frr.log
log timestamp precision 3
service integrated-vtysh-config
!
debug zebra events
debug zebra kernel
debug zebra rib
debug zebra nht
debug zebra nexthop
debug bgp keepalives
debug bgp neighbor-events
debug bgp nht
debug bgp updates in
debug bgp updates out
debug bgp zebra
debug bfd peer
debug bfd zebra
debug bfd network
!
router bgp 64512
 no bgp ebgp-requires-policy
 no bgp default ipv4-unicast
 no bgp network import-check
 neighbor A.B.241.5 remote-as 65080
 neighbor A.B.241.5 ebgp-multihop 255
 neighbor A.B.241.5 timers 30 90
 !
 address-family ipv4 unicast
  network 10.16.48.110/32
  network 10.16.48.111/32
  neighbor A.B.241.5 activate
  neighbor A.B.241.5 route-map A.B.241.5-in in
 exit-address-family
!
route-map A.B.241.5-in deny 20
!
route-map A.B.241.5-out permit 1
!
ip nht resolve-via-default
!
ipv6 nht resolve-via-default
!
line vty
!
end
/ # 
```

**Comment by fedepaol** [comment]
leaving this open until we don't bump the frr version

**Comment by fedepaol** [comment]
Closing this, frr bump happened. Hopefully this is fixed in 8.4.2 we are using now
